infra-as-code/bicep/modules/mgDiagSettings/README.md

```diff
@@ -0,0 +1,28 @@
+# Module: Enable Diagnostic Settings on Management Groups
+
+This module enables Diagnostic Settings on a management group to an existing Azure Log Analytics Workspace, normally deployed during the logging.bicep module.
+
+## Parameters
+
+The module requires the following input parameters. This parameter is passed from the mgDiagSettingsAll.bice module.
+
+| Parameter                             | Type   | Description                                                                                                                                                                          | Requirements                      | Example                                                                                 |
+| ------------------------------------- | ------ | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------ | --------------------------------- | --------------------------------------------------------------------------------------- |
+| parLawId    | string   | Id of the Log Analytics Workspace                                                             | Mandatory input, default: `false` | `false`                                                                                 |
+
+## Outputs
+
+The module will not generate any outputs.
+
+## Deployment
+
+This module requires being called with the  scope: managementGroup(mgId) function. This is done from within the orchestration module mgDiagSettingsAll.bicep module for all the management groups previously created and based on the input parameters for that parent module.
+The Diagnostic Settings resource will be named toLa but can be changed in the module if desired.
```

Comment:
Can you update this section to include the PowerShell and Azure CLI examples as per all the other modules please?

e.g. https://github.com/Azure/ALZ-Bicep/tree/main/infra-as-code/bicep/modules/subscriptionPlacement#deployment

Reply:
@jtracey93 , I had started working on that but because this module is only used when called from the other module (because the way in which the managementgroup(mg.id) function works I believe it cant be called from PowerShell or Azure CLI. That is what I meant on my note but lets discuss, I might be wrong.

Reply:
We discussed this 👍